src/cascadia/TerminalApp/JsonUtils.h

```diff
@@ -682,7 +682,7 @@ namespace TerminalApp::JsonUtils
     template<typename T, typename Converter>
     void SetValueForKey(Json::Value& json, std::string_view key, const T& target, Converter&& conv)
     {
-        json[key.data()] = conv.ToJson(target);
+        *json.demand(&*key.cbegin(), (&*key.cbegin()) + key.size()) = conv.ToJson(target);

```

Comment:
```suggestion
        // demand guarantees that it will return a value or throw an exception
        *json.demand(&*key.cbegin(), (&*key.cbegin()) + key.size()) = conv.ToJson(target);
```